app/services/github_service.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+require "erb"
+
+class GithubService
+  REPO = "department-of-veterans-affairs/appeals-support".freeze
+
+  def create_issue(record)
+    body = GithubIssueRenderer.new(username: record.username,
```

Comment:
maybe it'd be better to move this out to a place where we could test it

then the method would be: 
```
def create_issue(body)
end
```

Then we can do a test that the template is being filled out correctly